### DISCUSSION TITLE
Alterar Logo superior do StoryBook

### DISCUSSION THREAD

**Discussion by TaissaAlves** [storybookjs/storybook#28759] [Help] [answered]
### Summary

hello Storybook community

I'm new to StoryBook, I looked at the theme article and managed to successfully change the sideBar logo

However, I was unable to change the logo at the top of the page

![image](https://github.com/user-attachments/assets/108ee6d6-df4f-4caf-9406-809ab15fd2a9)

If anyone can help me, that would be great

thanks

### Additional information

import { addons } from '@storybook/addons';
import { create } from '@storybook/theming';

addons.setConfig({
  theme: create({ 
    base: 'light',
    brandImage: '/Logo.png'
  })
});

I did the code above in manager.js and changed the sideBar logo, but I need to change the navigation tab logo too

**Comment by yannbf** [comment, marked as answer]
Olá @TaissaAlves!

Welcome to the community! What you're looking for is a way to set a favicon. You can do that via the [.storybook/manager-head.html](https://storybook.js.org/docs/configure/user-interface/theming#css-escape-hatches) file.

This is an example that does that:
https://github.com/yannbf/mealdrop/blob/main/.storybook/manager-head.html

And here's how it looks like:
<img width="293" alt="image" src="https://github.com/user-attachments/assets/12cb0de0-4760-4088-b5c3-037914312e62">

@kylegach seems like we should improve our docs to elaborate more on the use cases for `manager-head.html`!
